HT6_3.py

```diff
@@ -0,0 +1,228 @@
+import os
+import random
+import datetime
+import normalization.HT6 as norm
+
+
+class Publication:
+    def __init__(self, publication_type: str, publication_body: str = "") -> None:
+        self.publication_type = publication_type
+        self.publication_body = publication_body
+
+
+class News(Publication):
+    def __init__(self, publish_datetime, publication_type, publication_body: str = "", location: str = "") -> None:
+        super().__init__(publication_type=publication_type, publication_body=publication_body)
+        self.publish_datetime = publish_datetime
+        self.location = location
+
+    def get_publish_date(self):
+        return self.publish_datetime.strftime('%Y-%m-%d %H:%M')
+
+    def get_news_location(self):

```

Comment:
I believe it is @staticmethod :) It is not referring to the object (self) inside the method